v2/tests/log_context_property_basic_types_ut/log_context_property_basic_types_ut.c

```diff
@@ -118,17 +118,17 @@ static void int32_t_to_string_with_NULL_value_fails(void)
 
 /* Tests_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_002: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).to_string shall call snprintf with buffer, buffer_length and format string PRId64 and pass in the values list the int64_t value pointed to be property_value. ]*/
```

Comment:
>format string PRId64 [](http://example.com/codeflow?start=157&length=20)

with some more umock-y like data structures, this format string could be asserted too. Right now, it is not asserted.

Reply:
Added this validation